src/ts/components/listbox.ts

```diff
@@ -13,8 +16,10 @@ import {DOM} from '../dom';
  *   </div
  * </code>
  */
+// TODO: change to extends container in v4
```

Comment:
Make it a little clearer (e.g. that the `ListSelector` should extend `Container`)